src/main/java/com/puppycrawl/tools/checkstyle/api/Violation.java

```diff
@@ -44,6 +44,8 @@
  * java.text.MessageFormat.
  *
  * @noinspection SerializableHasSerializationMethods, ClassWithTooManyConstructors
+ * @noinspectionreason we do not serialize this class
+ * @noinspectionreason immutable nature of class requires a bunch of constructors
```

Comment:
it become a problem to match easily inspection and reason.

should we do something like: 
```
 * @noinspection SerializableHasSerializationMethods, ClassWithTooManyConstructors
 * @noinspectionreason SerializableHasSerializationMethods - we do not serialize this class
 * @noinspectionreason ClassWithTooManyConstructors - immutable nature of class requires a bunch of constructors
```

Reply:
Yes, good idea

Reply:
Done